<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeeFeedBack</class>
 <widget class="QWidget" name="SeeFeedBack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeeFeedBack</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>717</width>
        <height>1500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>1500</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>机构反馈</string>
         </property>
         <widget class="QLabel" name="label_41">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>机构编号:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_emp_id">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>40</y>
            <width>281</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>XXXX</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>671</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_feedback">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>160</y>
            <width>661</width>
            <height>371</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_receive">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>1410</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>接收</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_emp_name">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>80</y>
            <width>371</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>XXXX</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_42">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>机构名称:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_43">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>550</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>附加：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="EnterViewLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>860</y>
            <width>481</width>
            <height>231</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="EnterViewLabel" name="label">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>580</y>
            <width>481</width>
            <height>231</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="EnterViewLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>1130</y>
            <width>481</width>
            <height>231</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EnterViewLabel</class>
   <extends>QLabel</extends>
   <header>enterviewlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
